app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -14,8 +14,22 @@
             app:title="@string/subscriptions_label"
             android:id="@+id/toolbar"/>
 
+    <TextView
+        android:id="@+id/feeds_filtered_message"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/toolbar"
+        android:background="?android:attr/selectableItemBackground"
+        android:gravity="start"
+        android:paddingStart="10dp"
+        android:paddingTop="4dp"
+        android:paddingEnd="10dp"
+        android:paddingBottom="6dp"
```

Comment:
Material design uses multiples of 8dp (and in some cases 4dp) for spacing. 6 and 10 are pretty uncommon numbers. Please align them with the material grid.